Bio/Align/stockholm.py

```diff
@@ -558,8 +568,8 @@ def format_alignment(self, alignment):
         #    alignment.column_annotations
         if alignment.column_annotations:
             for key, value in alignment.column_annotations.items():
-                feature = self.gc_mapping[key]
-                line = f"#=GC {feature}".ljust(start) + value + "\n"
+                feature = self.gc_mapping.get(key, key)
+                line = f"#=GC {feature} ".ljust(start) + value + "\n"
```

Comment:
Why the extra space after {feature}?

Reply:
A safety measure. If the (nonstandard) feature name is too long, there might be no whitespace between key and value. For the two letter well-behaving features the number of white space is not changed.

Reply:
OK